index.html

```diff
@@ -37,5 +37,174 @@ <h3 class="hey">Hey there <br> I'm Shadab</h3>
+                <ul class="usedTechs wc">
+                    <li class="tech twc">HTML</li>
+                    <li class="tech twc">Bootstrap</li>
+                    <li class="tech twc">Ruby</li>
+                </ul>
+                </div>
+            <div class="seeProject spwc">
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="workCard">
+            <div class="workCardContent">
+                <h3 class="recentWorkTitle wc">Profesional Art <br>Printing Data</h3>
+                <p class="recentWorkDesc wc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="usedTechs wc">
+                    <li class="tech twc">HTML</li>
+                    <li class="tech twc">Bootstrap</li>
+                    <li class="tech twc">Ruby</li>
+                </ul>
+                </div>
+            <div class="seeProject spwc">
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="workCard">
+            <div class="workCardContent">
+                <h3 class="recentWorkTitle wc">Profesional Art <br>Printing Data</h3>
+                <p class="recentWorkDesc wc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="usedTechs wc">
+                    <li class="tech twc">HTML</li>
+                    <li class="tech twc">Bootstrap</li>
+                    <li class="tech twc">Ruby</li>
+                </ul>
+                </div>
+            <div class="seeProject spwc">
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="workCard">
+            <div class="workCardContent">
+                <h3 class="recentWorkTitle wc">Profesional Art <br>Printing Data</h3>
+                <p class="recentWorkDesc wc">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <ul class="usedTechs wc">
+                    <li class="tech twc">HTML</li>
+                    <li class="tech twc">Bootstrap</li>
+                    <li class="tech twc">Ruby</li>
+                </ul>
+                </div>
+            <div class="seeProject spwc">
+                <a href="#">See Project</a>
+            </div>
+        </div> 
+    </section>
+    <section class="aboutme">
+        <h3 class="aboutmeTitle">About  me</h3>
+        <p class="aboutmeDesc recentWorkDesc wc">
+            Hello I’m a software developer! I can help you build a product , feature or website Look 
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="getResume seeProject">
+            <a href="#">Get My Resume</a>
```

Comment:
- [ ] Kindly let us add button interactions such as Active, Hover, and Pressed see design 👇 
![image](https://user-images.githubusercontent.com/55465133/129987112-9f70076d-6233-49ec-8e95-b746ed3076e5.png)

Please let us repeat this for all the buttons 👍